src/adjoint_common.jl

```diff
@@ -34,12 +34,14 @@ function adjointdiffcache(g::G, sensealg, discrete, sol, dgdu::DG1, dgdp::DG2, f
     prob = sol.prob
     u0 = state_values(prob)
     p = parameter_values(prob)
-    if p === nothing || p isa SciMLBase.NullParameters
+    if p === nothing || p === DiffEqBase.NullParameters()
```

Comment:
why the change to DiffEq? Should be SciML

Reply:
Got it